docs/deployment/kubernetes.md

```diff
@@ -0,0 +1,137 @@
+
+ 1. And then configure it in your application values within `anycable-go` namespace:
+
+    ```yaml
+    # values.yaml
+    anycable-go:
+      env:
+        # Assuming that Ruby RPC is available in K8s in the same namespace as anycable-rpc service (see next chapter)
+        anycableRpcHost: anycable-rpc:50051
+      ingress:
+        enable: true
+        path: /cable
+
+    # values/production.yaml
+    anycable-go:
+      env:
+        # Assuming that Redis is available in K8s in the same namespace as redis-anycable service
+        anycableRedisUrl: redis://:password@redis-anycable:6379/0
+      ingress:
+        acme: # if you're using Let's Encrypt
+          hosts:
+            - your-app.com
+    ```
+
+Read the [chart’s README][anycable-helm] for more info.
+
+## Ruby RPC server
+
+To run Ruby counterpart of Anycable which will handle connection authentication and execute your business logic we will need to create separate deployment and service for it.
+
+ 1. [**Deployment**](https://kubernetes.io/docs/concepts/workloads/controllers/deployment/) that will spin up required number of pods and handle rolling restarts on deploys
+
+    ```yaml
+    apiVersion: apps/v1
+    kind: Deployment
+    metadata:
+      name: anycable-rpc
+      labels:
+        component: anycable-rpc
+    spec:
+      replicas: 1
+      strategy:
+        type: RollingUpdate
+        rollingUpdate:
+          maxUnavailable: 1
+          maxSurge: 0
+      selector:
+        matchLabels:
+          component: anycable-rpc
+      template:
+        metadata:
+          labels:
+            component: anycable-rpc
+        spec:
+          containers:
+            - name: anycable-rpc
+              image: "{{ .Values.image.repository }}:{{ .Values.image.tag }}"
+              imagePullPolicy: IfNotPresent
+              command:
+                - bundle
+                - exec
+                - anycable
+                - !!str --rpc-host=0.0.0.0:50051
+              env:
+                - name: ANYCABLE_REDIS_URL
+                  value: "redis://:password@redis-anycable:6379/0" # Same as for anycable-go
+                  # or, better, place it into secrets for better security:
+                  # valueFrom:
+                  #   secretKeyRef:
+                  #     name: "anycable-go-secrets"
+                  #     key: anycableRedisUrl
+              resources:
+                limits:
+                  cpu: 500m
+                  memory: 700Mi
+                requests:
+                  cpu: 200m
+                  memory: 400Mi
+    ```
+
+ 1. [**Service**](https://kubernetes.io/docs/concepts/services-networking/service/) to connect anycable-go with RPC server.
+
+    ```yaml
+    apiVersion: v1
+    kind: Service
+    metadata:
+      name: anycable-rpc
+      labels:
+        component: anycable-rpc
+    spec:
+      selector:
+        component: anycable-rpc
+      type: ClusterIP
+      ports:
+        - port: 50051
```

Comment:
Should we extract port number to values?

Reply:
Yes, but it will make examples way harder to understand

Reply:
Let's add a comment like this:

```
# you should define these parameters in the values.yml file, we give them here directly for readability
```